src/zfstools/models.py

```diff
@@ -156,6 +156,7 @@ def parse_zfs_r_output(self, zfs_r_output, properties = None):
             assert 0, repr(properties)
 
         def extract_properties(s):
+            s = s.decode() if isinstance(s, bytes) else s
```

Comment:
Add an encoding UTF-8 to the parameter please.  Then I will merge unconditionally.